src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -19,15 +19,15 @@ const RenderChildDashboard = props => {
   };
 
   const handleJoinSquad = e => {
-    push('/child/join');
+    push('/join');
```

Comment:
We talked about removing the child route and you executed that task quickly. Great job as always Tony